modules/os2dagsorden/os2dagsorden_gs_conversion/os2dagsorden_gs_conversion.module

```diff
@@ -209,19 +197,16 @@ function os2dagsorden_gs_conversion_convert_single($fid, $uri) {
  *   The PDF version.
  */
 function os2dagsorden_gs_conversion_get_pdf_version($uri) {
+  // @CODEREVIEW: Why is the locale set here?
```

Comment:
we have had multiple problem with accessing files with Danish chars on some servers.
Adding this line removes those problems